x/mint/keeper/genesis.go

```diff
@@ -7,8 +7,8 @@ import (
 
 // InitGenesis initializes the x/mint store with data from the genesis state.
 func (k Keeper) InitGenesis(ctx sdk.Context, ak types.AccountKeeper, data *types.GenesisState) {
-	k.SetMinter(ctx, data.Minter)
 	// override the genesis time with the actual genesis time supplied in `InitChain`
+	k.SetMinter(ctx, data.GetMinter())
```

Comment:
I don't think this change does anything. Can we just have it as before to keep the PR diff low